services/product-service/src/init-db-script.ts

```diff
@@ -0,0 +1,16 @@
+import products from "./products.json";
+import dotenv from "dotenv";
+import { databaseService } from "./dependencies/dependencies";
+dotenv.config();
+
+const productsTable = process.env.TABLE_PRODUCTS;
+const stocksTable = process.env.TABLE_STOCKS;
+
+const initTables = async () => {
+  products.forEach(async ({ id, title, description, price, count }) => {
+    await databaseService.put(productsTable, { id, title, description, price });
```

Comment:
some improvement idea:  you already created a transaction-based product creation in DB service "createProduct". This 2 line could be replaced with that.